packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1152,6 +1158,10 @@ export function useGlobalStylesOutputWithConfig( mergedConfig = {} ) {
 		return !! getSettings().disableLayoutStyles;
 	} );
 
+	const blockContext = useContext( BlockContext );
+
+	const isTemplate = blockContext?.templateSlug !== undefined;
```

Comment:
If `blockContext` is undefined, should `isTemplate` be `undefined`, too, or is it safe for this to be `false`? I noticed the default values in the other functions is for `isTemplate` to be `true`, so wasn't sure.

Reply:
`isTemplate` should always be a boolean because we only use it in conditions, to check whether to output root padding or not. I set its default value to `true` for backwards compatibility purposes more than anything else.

Reply:
Sounds good. I could also imagine potentially in a follow-up, if folks wanted to force the value to `true`, we could add a prop to `useGlobalStylesOutputWithConfig` either supported within `mergedConfig`, or as an additional prop. Not to worry about for now, though!